demo/casc/packager-config.yml

```diff
@@ -0,0 +1,40 @@
+bundle:
+  groupId: "io.jenkins.tools.custom-war-packager.demo.casc"
+  artifactId: "jenkins-casc-demo"
+  vendor: "Jenkins project"
+  title: "Configuration-as-Code demo"
+  description: "Configuration-as-Code demo, produced by Custom WAR Packager"
+buildSettings:
+  docker:
+    base: "jenkins/jenkins:2.107.3"
+  #  tag: "jenkins/custom-war-packager-casc-demo"
```

Comment:
@MadsNielsen You can uncomment it and use the demo for quick packaging of Docker images with custom WAR file builds